e2e/playwright/file-tree.spec.ts

```diff
@@ -479,12 +466,8 @@ test.describe('Renaming in the file tree', () => {
       }
       const projectLink = page.getByText('Test Project')
       const projectMenuButton = page.getByTestId('project-sidebar-toggle')
-      const fileToRename = page
-        .getByRole('listitem')
-        .filter({ has: page.getByRole('button', { name: 'fileToRename.kcl' }) })
-      const renamedFile = page.getByRole('listitem').filter({
-        has: page.getByRole('button', { name: newFileName + FILE_EXT }),
-      })
+      const fileToRename = u.locatorFile('fileToRename.kcl')
```

Comment:
In many locations there were specific `pge.getByRole` calls, I've moved this to helper functions that still provide the `Locator` but now there are two functions to maintain for folder and file. This is a lot better.